src/initdsp.c

```diff
@@ -18,8 +18,8 @@
 #include <stdio.h>
 #include <unistd.h>
 
+#ifdef SUNDISPLAY
 #ifndef NOPIXRECT
-#ifndef DOS
 #ifdef XWINDOW
```

Comment:
This doesn't seem right, aren't  SUNDISPLAY and XWINDOW mutually exclusive?

Reply:
I thought so too! But ... I wasn't sure. There's no way to compile any of this, so I can take a best guess, but not sure what else.

Reply:
Okay, I've removed the whole chunk of includes there. Spot inspection makes it seem like it would be okay. (But of course, we'll never compile that code again anyway...)